### PR TITLE
updated questions.js

### PR DESCRIPTION
**Pull request by coverney**
https://trello.com/c/6JxvC3VV

### CODE REVIEW

File: data/questions.js [hunk 1]
```diff
@@ -3169,1336 +3163,1216 @@ export const questions = {
...
+            "question": "Crash County",
+            "id": "8V2H4hT4",
+            "answerType": "advancedDropDown",
             "display": [
-                "passenger"
+                "road"
             ],
-            "humanReadableId": "passenger-0"
+            "humanReadableId": "road-11",
+            "autoMethod": "county"
```
Comment (xieruishen) [inline data/questions.js:3650]: i think we still need to add a "answerOptions" field with an empty array [] for now just like the weather conditions advancedDropDown as currently the logic in line #42 of AdvancedDropDown.js is giving an error when trying to load Road (error: undefined is not an object (evaluating 'data.answerOptions.length')). Is it possible to also add a helper text for County saying "Choose a state first"? Thanks so much!

**Review by xieruishen — CHANGES_REQUESTED** [review]
state looks good. some minor issue with County that is failing the loading of Road in my comment ;) Thanks so much Cassandra!

**Comment by Enmoren** [comment]
Helpertext and tooltip parts look good to me!

**Review by xieruishen — APPROVED** [review]
looks good thanks so much! :)

**Pull request merged**